store/postgres/asset_repository_test.go

```diff
@@ -91,46 +95,54 @@ func (r *AssetRepositoryTestSuite) TearDownSuite() {
 	}
 }
 
-func (r *AssetRepositoryTestSuite) TestGetAll() {
-	// populate assets
-	total := 12
-	assets := []asset.Asset{}
-	for i := 0; i < total; i++ {
-		var typ asset.Type
-		var service string
-		if (i % 2) == 0 { // if even
-			typ = asset.TypeJob
-			service = "postgres"
-		} else {
-			typ = asset.TypeDashboard
-			service = "metabase"
-		}
+func (r *AssetRepositoryTestSuite) insertRecord() (assets []asset.Asset) {
+	filePath := "./testdata/mock-asset-data.json"
+	testFixtureJSON, err := ioutil.ReadFile(filePath)
+	if err != nil {
+		return []asset.Asset{}
+	}
+
+	var data []asset.Asset
+	if err = json.Unmarshal(testFixtureJSON, &data); err != nil {
+		return []asset.Asset{}
+	}
 
+	for _, d := range data {
 		ast := asset.Asset{
-			URN:       fmt.Sprintf("urn-get-%d", i),
-			Type:      typ,
-			Service:   service,
-			Version:   asset.BaseVersion,
-			UpdatedBy: r.users[0],
+			URN:         d.URN,
+			Name:        d.Name,
+			Type:        d.Type,
+			Service:     d.Service,
+			Description: d.Description,
+			Data:        d.Data,
+			Version:     asset.BaseVersion,
+			UpdatedBy:   r.users[0],
 		}
+
 		id, err := r.repository.Upsert(r.ctx, &ast)
 		r.Require().NoError(err)
 		r.Require().Equal(r.lengthOfString(id), 36)
 		ast.ID = id
 		assets = append(assets, ast)
 	}
 
+	return assets
+}
+
+func (r *AssetRepositoryTestSuite) TestGetAll() {
+	assets := r.insertRecord()
+
 	r.Run("should return all assets limited by default size", func() {
 		results, err := r.repository.GetAll(r.ctx, asset.Config{})
 		r.Require().NoError(err)
-		r.Require().Len(results, defaultGetMaxSize)
-		for i := 0; i < defaultGetMaxSize; i++ {
+		r.Require().Len(results, defaultGetAssetMaxSize)
+		for i := 0; i < defaultGetAssetMaxSize; i++ {
 			r.assertAsset(&assets[i], &results[i])
 		}
 	})
 
 	r.Run("should override default size using GetConfig.Size", func() {
-		size := 8
+		size := 3
```

Comment:
this test won't confirm anything, defaultGetAssetMaxSize is `3` and size is `3`, and the total assets in fixture is also `3`.